inspector/scripts/json-rpc-sync-check.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Dependencies: jq, curl
+
```

Comment:
how about declaring `KAIROS_JSON_RPC` and `KAIA_JSON_RPC` as const variable at the beginning of the script?
```suggestion
readonly KAIROS_JSON_RPC="http://100.93.31.29:8551"
readonly KAIA_JSON_RPC="http://100.75.43.49:8551"
```

and lets change `baobab` into kairos and `cypress` into `kaia` from the script

Reply:
done!